CMakeLists.txt

```diff
@@ -74,15 +74,18 @@ option(use_prov_client "Enable provisioning client" OFF)
 option(use_tpm_simulator "tpm simulator type of hsm used with the provisioning client" OFF)
 option(use_edge_modules "Enable support for running modules against Azure IoT Edge" OFF)
 option(use_custom_heap "use externally defined heap functions instead of the malloc family" OFF)
-option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" OFF)
+option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" ON)
+option(use_uhttp_upload_logging "set use_uhttp_upload_logging to ON to enable uhttp blob logging (only when use_uhttp_upload is also ON)" OFF)
+option(use_upload_to_blob_stress "set use_upload_to_blob_stress to run a blob stress test." OFF)
 set(compileOption_C "" CACHE STRING "passes a string to the command line of the C compiler")
 set(compileOption_CXX "" CACHE STRING "passes a string to the command line of the C++ compiler")
 set(linkerOption "" CACHE STRING "passes a string to the shared and exe linker options of the C compiler")
 
 set(use_prov_client_core OFF)
 
-# Temporary for now
-set(${use_uhttp_upload} ON)
+if(${use_uhttp_upload_logging})
```

Comment:
This should be in uhttp repo, AFAIK

Reply:
See above: this is for the SDK _using_ uhttp not the uhttp client.